apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SubscriberClientTest.cs

```diff
@@ -640,7 +640,7 @@ public void SlowUplinkThrottlesPull()
             }
         }
 
-        private class FakeSubscriberClient : SubscriberClient
+        private class FakeSubscriberClient : SubscriberServiceApiClient
```

Comment:
Consider renaming to FakeSubscriberServiceApiClient?

Reply:
Done, along with a couple of other similar renames.